04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+\chapter{Environment}
+\section{Foraging}
+
+\begin{definition} \label{def:Welfare}
+\textbf{Welfare} is the state of wealth, it can either be personal or collaborative (social).
```

Comment:
sentence fragment with the comma. Suggested rewrite: 

\textbf{Welfare} is defined as the state of wealth and can either be personal or collaborative (social).